### PR TITLE
Add: To present on the license

### PR DESCRIPTION
**Pull request by KarlGusta**
## Changes proposed

- Add 2021 - Present on the license.

## Check List (Check all the applicable boxes) 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] All new and existing tests passed.
- [ x] This PR does not contain plagiarized content.
- [ x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1193"></a>

### CODE REVIEW

**Review by ssensalo — COMMENTED** [review]
Isn't it preferable to have the year auto update?

**Comment by KarlGusta** [comment]
> Isn't it preferable to have the year auto-update?

If you have a way of doing that on a license file, I would love a review.

**Comment by schmelto** [comment]
> Isn't it preferable to have the year auto update?

I would not suggest this way due a year update is only needed if there are changes to the software (see [here](https://softwareengineering.stackexchange.com/questions/210472/is-renewal-of-mit-license-needed-on-github-at-the-beginning-of-each-year))

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Eddie Jaoude
+Copyright (c) 2021 - present Eddie Jaoude
```
Comment (schmelto) [inline LICENSE:3]:
```suggestion
Copyright (c) 2021 - 2022 Eddie Jaoude
```
Or in my understanding also correct could be
```suggestion
Copyright (c) 2021, 2022 Eddie Jaoude
```

**Review by krshdev — COMMENTED** [review]
My understanding is that the copyright year is the one the license was made in, and the years after are automatically included.

File: LICENSE (hunk 1 above)
Comment (Panquesito7) [reply LICENSE:3]: Maybe this too?

```suggestion
Copyright (c) 2021-2022 Eddie Jaoude
```

**Comment by KarlGusta** [comment]
I saw this on the vscode codebase license and thought of implementing it here.

**Comment by emmalearnscode** [comment]
From [copyright.com](https://www.copyrightlaws.com/copyright-symbol-notice-year/#:~:text=Which%20Year%20Should%20You%20Use,to%20the%20public%20without%20restriction.):

WHICH YEAR SHOULD YOU USE IN YOUR COPYRIGHT NOTICE?
The general rule is that the year to include in a copyright notice is the year of first publication of the work. First publication is when the work is made available to the public without restriction. This includes selling a book or leaving free copies of it in a public place or posting content on your blog. Publication does not include, for example, circulating copies internally to co-workers or sending copies of a book to a publisher.

For compilations or derivative works which incorporate previously published content, the year of first publication of the compilation or derivative work is sufficient.

WHAT ABOUT BLOGS AND WEBSITES?
New versions or editions of works should contain the publication date of the new version or edition. For constantly evolving websites and blogs that contain works published over several years, the notice may include a range of years (e.g., 2011–2021), starting from the date of the oldest published elements and ending with the date of the newest published elements.

So, it should probably be 2020 - 2022 if we choose to update this.

File: LICENSE (hunk 1 above)
Comment (schmelto) [inline LICENSE:3]:
```suggestion
Copyright (c) 2021 - 2022 Eddie Jaoude
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant, Maintainable

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**